ghost/README.md

```diff
@@ -0,0 +1,25 @@
+# Ghost
+
+Ghost is a Content Management System (CMS) for blogging. Think of it like the backend of a blog, but it's built so that it can be very easily accessed via an API. An example for NextJS is [here](https://ghost.org/docs/api/v3/nextjs/).
+
+Normally ghost is configured using a JSON file. But we can also set all the configuration through environment variables, where
+nested fields are accessed with a double underscore, just like in the Django ORM.
+
+Here are the secrets that need to be set:
+
+|               Key                       |  Description                                                |
+|-----------------------------------------|-------------------------------------------------------------|
+| `AWS_ACCESS_KEY_ID`                     |                                                             |
+| `AWS_DEFAULT_REGION`                    |                                                             |
+| `AWS_SECRET_ACCESS_KEY`                 |                                                             |
+| `GHOST_STORAGE_ADAPTER_S3_PATH_BUCKET`  |                                                             |
```

Comment:
can we get an example of this

Reply:
Not 100% sure what each of them mean, it has to do with IAM roles in AWS, was going to ask you or @ArmaanT to fill them in

Reply:
OK I can fill them in.